src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFADetailsPage.java

```diff
@@ -27,25 +26,21 @@
     @FindBy(how = How.CLASS_NAME, using = "topName")
     private WebElement newAccountLabel;
 
-    /**
-     * Constructor.
-     */
-    public SFADetailsPage() {
-        super();
-        PageFactory.initElements(this.webDriver, this);
-    }
 
     /**
      * Method to get edit button.
+     *
      * @return the edit button as WebElement.
      */
     public WebElement getEditButton() {
-        CommonActions.scrollPage(this.webDriver, this.editButton);
+
+        // CommonActions.scrollPage(this.webDriver, this.editButton);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)